.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
FYI: you can do `[ "$TRAVIS_OS_NAME" == "osx" ] && brew update` - imho it's a lot easier to read & one can directly jump to the end of the command